samples/subsys/ipc/openamp/sample.yaml

```diff
@@ -4,6 +4,7 @@ sample:
 tests:
   sample.ipc.openamp:
     filter: dt_chosen_enabled("zephyr,ipc") and dt_chosen_enabled("zephyr,ipc_shm")
+      and not (CONFIG_SOC_ESP32_APPCPU or CONFIG_SOC_ESP32S3_APPCPU)
```

Comment:
is this really an espressif issue or are we going to see more filters like this for other platforms ? Maybe the original filter is too open and we will start seeing more platforms failing here...

Reply:
`openamp` had some specific platforms defined before as you can see in https://github.com/zephyrproject-rtos/zephyr/pull/85997/files#diff-f039611c7e2d4ece83aafb2ef58617094bb8a4d569a8a06ddb182c45443692d9L6
That change just increased the filter as you see there. I could not figured out yet a way to exclude all esp32 `appcpu` boards from current filter.

Reply:
> is this really an espressif issue or are we going to see more filters like this for other platforms ? Maybe the original filter is too open and we will start seeing more platforms failing here...

@nashif I have do a full build on all with '-l' see here. https://github.com/zephyrproject-rtos/zephyr/pull/87165#issuecomment-2728053453